metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/enrichment_commands.py

```diff
@@ -136,26 +136,37 @@ def init_kafka_acls(self):
 
         self.set_kafka_acl_configured()
 
-    def start_enrichment_topology(self):
+    def start_enrichment_topology(self, env):
         Logger.info("Starting Metron enrichment topology: {0}".format(self.__enrichment_topology))
-        start_cmd_template = """{0}/bin/start_enrichment_topology.sh \
-                                    -s {1} \
-                                    -z {2}"""
-        Logger.info('Starting ' + self.__enrichment_topology)
-        Execute(start_cmd_template.format(self.__params.metron_home, self.__enrichment_topology, self.__params.zookeeper_quorum),
-                user=self.__params.metron_user)
+
+        if self.is_topology_active(env):
```

Comment:
Shouldn't we only be starting the topology if it's not active?  I think this condition needs to be negated.